content/commands/jx_create_cluster_minikube.md

```diff
@@ -31,8 +31,13 @@ jx create cluster minikube [flags]
       --cleanup-temp-files                  Cleans up any temporary values.yaml used by helm install [default true] (default true)
       --cloud-environment-repo string       Cloud Environments Git repo (default "https://github.com/jenkins-x/cloud-environments")
   -c, --cpu string                          Number of CPUs allocated to the Minikube VM. Defaults to 3.
+<<<<<<< HEAD
```

Comment:
Merge Conflicts

Reply:
Great thanks for catching that.  Just fixed it.